news/tasks.py

```diff
@@ -491,7 +487,7 @@ def send_transactional_messages(data, transactionals):
             lang_code = 'en'
 
         msg_id = mogrify_message_id(tm.vendor_id, lang_code, 'H')
-        send_message.delay(msg_id, email, email, 'H')
+        send_message.delay(msg_id, email, email)
```

Comment:
I've left it for now because when we do the deploy there could be tasks on the queue that have that extra argument. Can be removed later. I should add a TODO.